node/tests/SharedTests.ts

```diff
@@ -2527,6 +2527,228 @@ export function runBaseTests<Context>(config: {
         },
         config.timeout,
     );
+
+    // Set command tests
+
+    async function classicSet(client: BaseClient) {
```

Comment:
It is already being tested in "set and get flow works" which calls GetAndSetRandomValue